crates/bevy_pbr/src/atmosphere/sky_view_lut.wgsl

```diff
@@ -26,47 +23,20 @@
 fn main(@builtin(global_invocation_id) idx: vec3<u32>) {
     let uv = vec2<f32>(idx.xy) / vec2<f32>(settings.sky_view_lut_size);
 
-    let r = view_radius();
+    // let world_pos = vec3<f32>(0.0, 0.0, 0.0); // get_view_position();
+    let r = view_radius_constant();
+    let world_pos = vec3<f32>(0.0, r, 0.0);
```

Comment:
should this be get_view_position?

Reply:
Good question, I have a long answer to this. No, the sky view lut calculation assumes a non-spherical coordinate system because of the way it is parameterized. This is a limitation of this optimization technique. The result is that the horizon may be out of alignment for renderings that rely on the sky view lut. at the present we use it for environment maps, and if the raymarching is turned on this out of alignment of the horizon is not noticeable at high altitudes. That's thanks to the spherical coordinate system that the raymarched rendering introduces.